addons/website/static/src/snippets/s_tabs/options.js

```diff
@@ -11,6 +11,14 @@ options.registry.NavTabs = options.registry.MultipleItems.extend({
      */
     start: function () {
         this._findLinksAndPanes();
+        // To make sure the selection remains bounded to the active tab, each
+        // tab is made non editable while keeping its nested oe_structure
+        // editable. This avoids having a selection range span over all
+        // further inactive tabs when using Chrome.
+        for (const tabEl of this.$tabPanes) {
+            tabEl.setAttribute('contenteditable', false);
+            tabEl.querySelector(':scope > .oe_structure').setAttribute('contenteditable', true);
+        }
```

Comment:
@bso-odoo Not a fan of this fix:
- `start` is executed way after a selection on the snippet could have been made in case the snippet editor instance is not created yet. Messing with `contenteditable` in there is a thing of the past IMO.
- The editor team made several functions which allow to mark contenteditable true/false nodes. I think those are called at editor load but also when clicking on new areas or something like that. See
 ```javascript
getReadOnlyAreas: () => [],
getContentEditableAreas: () => [],
```
We already used it for some fixes like `const $extraEditableZones = $editableSavableZones.find('.s_company_team .o_not_editable img');` (although we should still review this system IMO...).
- Note that for the tabs themselves, we could probably use the `o_not_editable` class directly but the mentioned editor teams functions should do the trick as well (and this is more stable than having to patch the DOM).